app.js

```diff
@@ -1,8 +1,62 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
 // import functions and grab DOM elements
+const case1Button = document.getElementById('case1-button');
+const case2Button = document.getElementById('case2-button');
+const case3Button = document.getElementById('case3-button');
 
+const case1Container = document.getElementById('case-container');
+const case2Container = document.getElementById('case2-container');
+const case3Container = document.getElementById('case3-container');
+
+const totalEl = document.getElementById('total');
+const lossesEl = document.getElementById('losses');
+const winsEl = document.getElementById('wins');
+
+const cases = ['case', 'case2', 'case3'];
 // let state
+let totalScore = 0;
+let totalWins = 0;
 
 // set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+case1Button.addEventListener('click', () => {
+    const caseSelect = Math.floor(Math.random() * 3);
+    const answer = cases[caseSelect];
+    handleGuess(answer, 'case1');
+});
+
+case2Button.addEventListener('click', () => {
+    const caseSelect = Math.floor(Math.random() * 3);
+    const answer = cases[caseSelect];
+    handleGuess(answer, 'case2');
+});
+
+case3Button.addEventListener('click', () => {
+    const caseSelect = Math.floor(Math.random() * 3);
+    const answer = cases[caseSelect];
+    handleGuess(answer, 'case3');
+});
+
+function handleGuess(correctSpot, userGuess) {
+    // reset the styles
+    case1Container.classList.remove('money');
+    case2Container.classList.remove('money');
+    case3Container.classList.remove('money');
+    // then increment the guesses
+    if (userGuess === correctSpot) {
+        totalWins++;
+        totalScore++;
```

Comment:
Similar to the half-baked, you'll want your `totalScore` to increment regardless of the outcome of the guess, and adding the `textContent` should be handled outside of your conditional statement.